roles/rke2_common/tasks/cis-hardening.yml

```diff
@@ -40,7 +40,7 @@
         remote_src: yes
         mode: 0600
       register: sysctl_operation_tarball
-      when: ansible_facts['os_family'] != 'RedHat' or rke2_binary_tarball_check.stat.exists
+      when: (ansible_facts['os_family'] != 'RedHat' and ansible_facts['os_family'] != 'Rocky') or rke2_binary_tarball_check.stat.exists
```

Comment:
I tried to comment on this one too but it appears Github had some issues earlier today and commenting stopped working for a while right when I was trying to type. 
```
when: >-
  (ansible_facts['os_family'] != 'RedHat' and
  ansible_facts['os_family'] != 'Rocky') or
  rke2_binary_tarball_check.stat.exists
```
to pass linter

Reply:
Done